app/components/provider_interface/status_box_components/conditions_not_met_component.rb

```diff
@@ -4,8 +4,8 @@ class ConditionsNotMetComponent < ActionView::Component::Base
       include ViewHelper
       attr_reader :application_choice
 
-      def initialize(application_choice:)
-        @application_choice = application_choice
+      def initialize(args)
```

Comment:
Were we going leave this (and its like) as simply `application_choice:`?

Reply:
Yep, I've restored application_choice as an argument.